packages/website/start.js

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env node
+const { spawnSync } = require('child_process');
+const generatePages = require('./generate-pages');
+
```

Comment:
Can we move pulling and parsing config into `run.js`, so that there is a source of truth for getting a config between commands?

Reply:
will do in AFP-192